nnpdf_data/nnpdf_data/commondata/CMS_WPWM_7TEV_MUON/metadata.yaml

```diff
@@ -40,8 +40,8 @@ implemented_observables:
         label: $\eta$
         units: ''
       M2:
-        description: W boson Mass
-        label: $M^2$
+        description: W boson mass squared
```

Comment:
You forgot the `M2` of the name of the dict in this file.

Reply:
Thanks for that. Updated!

Reply:
Btw, I think you will need to update also filter rules to use the right `m_w` etc

Reply:
Yes, indeed.